Azure/sap-hana
# Add Loadbalancer Frontend IP to the output JSON file

## Problem

As part of #338 and the future RHEL work, the Load Balancer Frontend IP is required by the STONITH clustering configuration.

## Description

This PR adds the loadbalancer frontend IP to the `output.json` for use by Ansible.

## Pre-Requisites

None

## Commitment to Test

- Testing takes 15 minutes
- Reviewing 1 file

## Test Instructions/Acceptance Criteria

:hand: The following testing was performed in [PR #23 on the Centiq Fork](https://github.com/Centiq/sap-hana/pull/23)

1. Review the code changes
   - [x] Code changes are understandable and acceptable
   - [x] Pipeline templates are updated
   - [x] Relevant template samples are updated
1. Ensure your working directory is clean:\
   `util/terraform_v2.sh clean`
   `y`
1. Initialise the terraform modules:\
   `util/terraform_v2.sh init`
1. Update the resource group in the `single_node_hana` template (make sure this is personalised/not going to conflict with any other resources), e.g.:\
   `util/set_resource_group.sh "pc-lb-tf-test-rg" "single_node_hana"`
1. Build the infrastructure:\
   `util/terraform_v2.sh apply single_node_hana`
1. Using the Azure Portal, navigate to your resource group and note the `Private IP address` of the Load Balancer
1. Using the output from the above, SSH to the RTI instance:\
   `ssh azureadm@<IP_ADDRESS>`
1. Check the `output.json` file for the azure configuration:\
   `cat output.json | |python -m json.tool | grep -A2 loadbalancer`
   - [x] The `frontend_ip` value matches the above noted value

## Post-Testing

:hand: Make sure you delete the resource group created for this test after completion.
